user-manual/transfer/import-metadata.rst

```diff
@@ -355,44 +355,208 @@ The ``rights.csv`` file is parsed by the job "Load Rights" within the
-          <mets:fptr FILEID="file-a96caae3-d15b-4dcd-851e-0369805b0751"/>
-        </mets:div>
+     <mets:div TYPE="track" LABEL="Complete documentary">
+       <mets:div LABEL="Introduction" ORDER="1">
+         <mets:fptr FILEID="file-a47cee9a-7508-4189-aa21-76ab3d02e2a2" CONTENTIDS="objects/ferdinand_short_2017_01_27.mp3">
+           <mets:area FILEID="file-a47cee9a-7508-4189-aa21-76ab3d02e2a2" CONTENTIDS="objects/ferdinand_short_2017_01_27.mp3" BEGIN="00:00:00" END="00:00:17" BETYPE="TIME"/>
+         </mets:fptr>
+       </mets:div>
+       <mets:div LABEL="Outro" ORDER="2">
+         <mets:fptr FILEID="file-a47cee9a-7508-4189-aa21-76ab3d02e2a2" CONTENTIDS="objects/ferdinand_short_2017_01_27.mp3">
+           <mets:area FILEID="file-a47cee9a-7508-4189-aa21-76ab3d02e2a2" CONTENTIDS="objects/ferdinand_short_2017_01_27.mp3" BEGIN="00:00:18" END="00:01:13" BETYPE="TIME"/>
+        </mets:fptr>
+      </mets:div>
     </mets:div>
    </mets:structMap>
 
+Here is another example of a custom METS structural map for a simple book. The
+transfer's ``objects/`` directory contains all of the digital files used to
+compile the book (e.g. cover.jpg, inside_cover.jpg, page_01.jpg, etc.) The
+transfer's ``metadata/`` directory contains the following ``mets_structmap.xml``
+file to define the structure of the book. Upon import, Archivematica will add a
+unique ID to the ``structMap`` element and update all the ``FILEID`` attributes
+to match the UUID value for these files in the Archivematica AIP.
+
+.. code:: xml
+
+   <?xml version="1.0" encoding="utf-8"?>
+     <mets:mets xmlns:mets="http://www.loc.gov/METS/">
+       <mets:structMap TYPE="logical">
+         <mets:div TYPE="book" LABEL="How to create a hierarchical book">
+           <mets:div TYPE="page" LABEL="Cover">
+             <mets:fptr FILEID="cover.jpg" CONTENTIDS="objects/cover.jpg"/>
+           </mets:div>
+           <!-- cover -->
+           <mets:div TYPE="page" LABEL="Inside cover">
+             <mets:fptr FILEID="inside_cover.jpg" CONTENTIDS="objects/inside_cover.jpg"/>
+           </mets:div>
+           <!-- inside cover -->
+           <mets:div TYPE="chapter" LABEL="Chapter 1">
+             <mets:div TYPE="page" LABEL="Page 1">
+               <mets:fptr FILEID="page_01.jpg" CONTENTIDS="objects/page_01.jpg"/>
+             </mets:div>
+             <mets:div TYPE="subchapter" LABEL="Subchapter 1.1">
+               <mets:div TYPE="page" LABEL="Page 2">
+                 <mets:fptr FILEID="page_02.jpg" CONTENTIDS="objects/page_02.jpg"/>
+               </mets:div>
+               <mets:div TYPE="page" LABEL="Page 3">
+                 <mets:fptr FILEID="page_03.jpg" CONTENTIDS="objects/page_03.jpg"/>
+               </mets:div>
+               <mets:div TYPE="page" LABEL="Page 4">
+                 <mets:fptr FILEID="page_04.jpg" CONTENTIDS="objects/page_04.jpg"/>
+               </mets:div>
+               <mets:div TYPE="subchapter" LABEL="Subchapter 1.2">
```

Comment:
It might make sense to update this with the change made in sample data to bring the two sub-chapters level, but it might not particularly bother you, if so, no problem here either.

Reply:
yes, good point. done.